src/lib.rs

```diff
@@ -497,6 +597,8 @@ impl Octocrab {
     }
 }
 
+const MAX_RETRIES: u32 = 3;
```

Comment:
Can we move this to the very top of the file?

Reply:
Sure